src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -632,7 +634,8 @@ protected virtual void MarkUserDependency (MemberReference context, CustomAttrib
 			if (MarkDependencyField (td, member, new DependencyInfo (DependencyKind.PreservedDependency, ca)))
 				return;
 
-			_context.LogMessage (MessageImportance.High, $"Could not resolve dependency member '{member}' declared in type '{td.FullName}'");
+			_context.LogMessage (MessageContainer.CreateWarningMessage ($"Could not resolve dependency member '{member}' declared in type '{td.FullName}'",
+				2005, origin: MessageOrigin.TryGetOrigin (context.Resolve (), 0)));
```

Comment:
I'm not sure we should report IL offset 0 for the attribute - if we can't get the line number of the attribute itself, I would rather just mention that it came from the attribute on the method.